app/serializers/note_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class NoteSerializer < ActiveModel::Serializer
+  attributes :title, :body, :created_at, :updated_at
+
+  has_many :tags, through: :taggings
```

Comment:
Don't need the `through:` bit in the serializer. `has_many` means a different thing here in a serializer (just means: There are many of these, make it an array)